cmake/FindCUDACupti.cmake

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
we should maybe have something like what's in [Assertions](https://github.com/oneapi-src/unified-runtime/blob/c259937a690afb0a2a64446aa06310f95e6c411a/cmake/Assertions.cmake#L1) if this is lifted from LLVM

Reply:
It's from the SYCL specific cmake folder rather than upstream LLVM so I think the copyright is fine, but I can add a note here anyway